src/main/java/digital/slovensko/autogram/core/eforms/dto/EFormAttributes.java

```diff
@@ -52,6 +55,9 @@ private static EFormAttributes build(String identifier, String transformation, S
                 throw new EFormException("Chýba identifikátor", "Identifikátor je povinný atribút pre XML Datacontainer");
         }
 
+        if (EFormUtils.isOrsrUri(identifier))
+            embedUsedSchemas = true;
```

Comment:
Ja by som to radšej takto overridol. Je tu riziko, že v ďalekej budúcnosti prejdú na referencie, ale v tom momente by aj tak prešli na data.gov.sk/slovensko.sk formuát, takže je to jedno.